### DISCUSSION TITLE
Missing files in problemtype folder?

### DISCUSSION THREAD

**Discussion by mikael-onsilva** [Anura3D/Anura3D_OpenSource#219] [Q&A] [answered]
Hello guys, I've noticed that some files referenced in the tutorial manual does not appear in the problemtype folder [here](https://github.com/Anura3D/Anura3D_OpenSource/tree/main/src/GiD_Problemtype/Anura3D_2023.gid) and aren't created when the code is built.

The missing files are "_Anura3D_2023.mat_" and "_Anura3D_2023.prb_", as refered in #186 and mentioned on #126.

Are these files not necessary anymore and the Figure 2.13 isn't updated, or these missing files will cause some errors?

In this same Figure 2.13, folder _exec_ should exist a _libiomp5md.dll_ file, that isn't there as well. Is this an error in the figure also?

Thanks in advance.

**Comment by mikael-onsilva** [comment]
I've noticed these files were there in the 2022 version and weren't created since the 2023 release on Jun 30th.

**Comment by gdicarluccio** [comment, marked as answer]
Thank you for your feedback! Figure 2.13 in the manual is not updated with the new structure of the problem type. We will fix this in the next release.
